components/sql/src/test/scala/pl/touk/nussknacker/sql/utils/WithPostgresqlDB.scala

```diff
@@ -0,0 +1,58 @@
+package pl.touk.nussknacker.sql.utils
+
+import com.dimafeng.testcontainers.{ForAllTestContainer, PostgreSQLContainer}
+import org.hsqldb.jdbcDriver
+import org.scalatest.BeforeAndAfterAll
+import org.testcontainers.utility.DockerImageName
+import scala.jdk.CollectionConverters._
+
+import java.sql.{Connection, DriverManager}
+import java.util.UUID
+
+trait WithPostgresqlDB {
```

Comment:
Cannot we just move `pl.touk.nussknacker.test.base.db.WithPostgresDbTesting` to the `test-utils` module and reused it in designer and in sql component?

Reply:
Rozumiem, że chodzi o połączenie tego treita z traitem WithTestPostgresDb z desginera. Moim zdaniem to nie da za dużo, bo część wspólna ich to jest tylko stworzenie test containera i wyłuskanie  z niego danych do połaczenia się z bazą danych. Czyli dość mało. A też te traity teraz robią jakieś inne rzeczy